docs/index.rst

```diff
@@ -32,6 +32,7 @@ Contents
    :maxdepth: 3
 
    installing
+   flask
```

Comment:
If we are to include `flask.rst` in the documentation, I would suggest to move it below `quickstart.ipynb`. That file contains some more basic examples.

Reply:
Whoops, I meant to put it under `quickstart` thanks for the catch